### ISSUE TITLE
Order Payment Total - wrong formatting when using different decimal factor

### ISSUE THREAD

**Issue by MatejaPodravac** [coreshop/CoreShop#2699]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? | no
| BC Break report? | no
| RFC?             | no

On the order overview page in Pimcore backend in the summary section order payment total is not displayed correctly. For example, total is 44€ and payment total stored is 4400. 

Configuration for fraction digits:
```
core_shop_currency:
    money_decimal_precision: 2
    money_decimal_factor: 10000
```

coreshop.util.format.currency relies completely on the defined decimal factor which results in payment total not being displayed correctly. Instead of defined decimal factor, it should rely on the factor passed from OrderController `getSummary` for every summary item. Only if it's not passed use defined fraction digits.

![image](https://github.com/user-attachments/assets/feff9b77-e561-4efa-9d6d-aab26e0b8036)
![image](https://github.com/user-attachments/assets/edd15b71-d963-4f5b-b601-c4882bbf8552)

**Comment by MatejaPodravac** [comment]
@dpfaffenbauer do you plan to include this fix in the 4.0.10 release?

**Comment by dpfaffenbauer** [comment]
@MatejaPodravac it is, every change in 3.2.x gets merged up to 4.0.x and next
